issue_04/script.js

```diff
@@ -0,0 +1,28 @@
+const contentsArray = [{
+    to: "bookmark.html",
+    img: "1.png",
+    alt: "画像1",
+    text: "ブックマーク"
+}, {
+    to: "message.html",
+    img: "2.png",
+    alt: "画像2",
+    text: "メッセージ"
+}];
+
+const ul = document.createElement('ul');
+const fragment = document.createDocumentFragment();
+
+for (const item of contentsArray) {
+    const li = document.createElement('li');
+    const a = document.createElement('a');
+    const img = document.createElement('img');
+
+    a.textContent = item.text;
+    a.href = item.to;
+    img.src = item.img;
+    img.alt = item.alt;
+
+    fragment.appendChild(li).appendChild(a).insertAdjacentElement('afterbegin', img);
+}
+document.body.insertAdjacentElement('afterbegin', ul).appendChild(fragment);
```

Comment:
Question

Why did you omit HTML ul tag and add DOM to the body tag?

Reply:
I thought the task was to build the DOM in JavaScript, including the ul tag.

Reply:
Thank you for the answer.
As a code, I thought it would be easier to read if you installed ul like in issue 3.